app/views/recommended_links/show.html.erb

```diff
@@ -28,6 +28,25 @@
   padding: true
 } %>
 
+<div id="recommended_link_preview">
+  <%= render "govuk_publishing_components/components/document_list", {
+    remove_underline: true,
+    items: [
+      {
+        link: {
+          text: @recommended_link.title,
+          path: @recommended_link.link,
+          description: @recommended_link.description,
+        },
+        metadata: {
+          public_updated_at: Time.now,
```

Comment:
Ah, I see. This would normally be the date that the content was publicly updated.
Not all content displays a date, so you could probably get away with removing it.

<img width="684" alt="Screenshot 2024-04-25 at 17 39 38" src="https://github.com/alphagov/search-admin/assets/5793815/cf47670d-ec42-44fd-a937-7e98b92015bc">

It's the link information that's important I think rather than the date.
However this is not a blocker. It's also fine to leave a dummy date in as an example of metadata. It might be best to check with the people who will using this tool.